ZenPacks/zenoss/LinuxMonitor/LinuxService.py

```diff
@@ -0,0 +1,25 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+
+"""LinuxService module.
+
+All custom behavior for Linux OS Service is defined here.
+
+"""
+
+
+from . import schema
+
+
+class LinuxService(schema.LinuxService):
```

Comment:
Still a question for me whether to inherit it from OSComponent or use zenpacklib.Component as you can see below